Add new map "Mars Colony" by Anonomous

AnonomousAccount opened the pull request:
**THIS MAP HAS ALREADY BEEN APPROVED BY MAP TESTERS**

_A Colony on Mars_ features dozens of structures scattered across a martian landscape.

The old PR of this map was closed because I made a mistake with github, but now has been reopened. All issues and suggestions posed by map testers have been adressed. This map is now in need of **manager review**

AnonomousAccount commented:
Still waiting for manager review @olliy1or @HobbitPower @MoreKat

farooqkz commented:
I saw in my GH notifications that the PR was closed I was like "wait what? it was a good map..." and now I realized you opened another PR :)

HobbitPower commented:
@AnonomousAccount 
* Is this intentional? ![image](https://github.com/user-attachments/assets/c6e8a2d6-6782-4619-b536-815ef63f2c1d) 
* got this error again: 
![image](https://github.com/user-attachments/assets/cc44009a-4051-4d0f-acbe-5e48fd79fd20)

AnonomousAccount commented:
where did that hole come from lol

AnonomousAccount commented:
I'm not seeing that hole, or getting the barrier issue
![screenshot_20240802_113723](https://github.com/user-attachments/assets/eff471f8-a847-4dfb-90e7-9ccc43ca1ca2)

AnonomousAccount commented:
These new files no longer give me the barrier issue 👍

AnonomousAccount commented:
Ok, these are Soul's barrier files that were working

AnonomousAccount commented:
![screenshot_20240923_181850](https://github.com/user-attachments/assets/52c35757-2fc8-4451-9841-c2a23140e5ef)
The 4 mese ores on blue side are in this rocket. While they are a bit hard to find, these 4 on either side are more meant to be sorta bonus mese, since the bulk of it is in the rovers.

AnonomousAccount commented:
![screenshot_20240923_183247 copy](https://github.com/user-attachments/assets/1e6bfc1a-5d2e-477c-91a0-dc5b6e8b9cb7)
![screenshot_20240923_183251 copy](https://github.com/user-attachments/assets/39d19a83-54d0-4cd2-a998-c0850c0ca05b)
This area is now also breakble

AnonomousAccount commented:
Blueberries are now balanced, 14 per side. I don't think the mushrooms are too big of a deal, they are alot less powerful,

HobbitPower commented:
> I don't think the mushrooms are too big of a deal, they are alot less powerful,
* This mostly referred to the fact that the side had more blueberry bushes **and** additionally also mushrooms. So a double advantage. Blueberry bushes are now evenly distributed, so that's fine for me. 

* I see you didn't implement Kat's suggestion and thus didn't add coal ores or coal blocks? At least you can find wood in the chests.

* Your screenshot could also show the whole map, as in this example here: 
![screenshot](https://github.com/user-attachments/assets/eace98c3-696d-4dc3-aad0-89a88e0336d0)

AnonomousAccount commented:
Why would we want players to tunnel lol

farooqkz commented:
Half the fun of CTF is tunneling XD

AnonomousAccount commented:
INIT.LUA
A new block, "desert iron" has been added. It's basicaly iron ore, the only changes being that the texture now uses the desert cobble texture instead of the stone texture, making it fit into the landscape much better. Desert Iron also directly drops steel ingots.

AnonomousAccount commented:
MAP.MTS
- Switched old ores for the new "desert iron"
- Changed a dozen blocks to make one part of the landscape smoother
- Removed a patch of indestructible desert cobble that was accidentally put in instead of normal desert cobble

HobbitPower commented:
@AnonomousAccount Personally, I think you could have done without “desert iron”. Adding iron and coal ore would have been enough. 
Note that usually @LoneWolfHT  checks init.lua files for functionality etc. But it seems to work just fine.

LoneWolfHT commented:
> INIT.LUA A new block, "desert iron" has been added. It's basicaly iron ore, the only changes being that the texture now uses the desert cobble texture instead of the stone texture, making it fit into the landscape much better. Desert Iron also directly drops steel ingots.

I'd prefer a PR to the capturetheflag repo that adds variants in this spirit for all maps to use

AnonomousAccount commented:
Are you still fine with this iron directly dropping ingots? Or would you prefer I use normal desert iron

AnonomousAccount commented:
Or we could add direct-to-steel iron to CTF?

LoneWolfHT commented:
> Are you still fine with this iron directly dropping ingots? Or would you prefer I use normal desert iron

I feel like that's a little too much, in classic mode at least

AnonomousAccount commented:
Removed Desert Iron (Just normal iron)

AnonomousAccount commented:
Map should comply with all requests now

ClothierEdward commented:
I think this map is almost perfect. I really like the theming with being a colony in Mars. I like all the buildings, especially the biosphere in the center and the greenhouse on the blue side. The only change I'd make is to add drills to both sides of the map with small holes that lead to where the iron is. Other than, this is an excellent map!

AnonomousAccount commented:
Map was merged on JMA.

From some real-game play experience, the bases work well, and are easy enough to infiltrate. The map seems to be large enough for enjoyable gameplay (with ~20 people)

AnonomousAccount commented:
Note: I may have forgotten team chests in this version, I'll check later
